e2e/testcases/helm_sync_test.go

```diff
@@ -136,6 +137,131 @@ func TestPublicHelm(t *testing.T) {
 	}
 }
 
+// TestHelmWatchConfigMap can run on both Kind and GKE clusters.
+// It tests that helm-sync properly watches ConfigMaps in the RSync namespace and pulls updated values.
+func TestHelmWatchConfigMap(t *testing.T) {
```

Comment:
Looks like this only tests RootSync, we need test cases for RepoSync as well.

Side note: I expected to see the ns-reconciler ClusterRole updated with permission to read ConfigMaps. Is the helm-sync container for RepoSync able to read the ConfigMap currently?

Reply:
Will add tests for RepoSync. I think you're right that we'll need to add ConfigMaps to the ns-reconciler ClusterRole. I'm also working on adding tests for error cases